test/routes/index.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable global-require */
+
+'use strict';
+
+const should = require('should');
+
+describe('routes', () => {
+  describe('stripUnknown', () => {
+    const originalEnv = Object.assign({}, process.env);
+
+    beforeEach(() => {
+      delete require.cache[require.resolve('../../routes')];
+    });
+
+    afterEach(() => {
+      delete require.cache[require.resolve('../../routes')];
+      process.env = originalEnv;
```

Comment:
Nice touch, I actually came here to suggest it and :boom: you had it in already :wink: